sentry-ruby/lib/sentry/rake.rb

```diff
@@ -1,30 +1,31 @@
 require "rake"
 require "rake/task"
 
-module Rake
-  class Application
+module Sentry
+  module Rake
+    module Application
+      def display_error_message(*)
```

Comment:
shouldn't this method take `ex` as the argument?

Reply:
yes, we need to do that to make it working with ruby 2.4, let me fix that